torch/_inductor/utils.py

```diff
@@ -894,6 +894,16 @@ def is_cpu_device(inputs):
     )
 
 
+def get_sympy_Expr_dtype(val):
```

Comment:
```suggestion
def get_sympy_Expr_dtype(val : sympy.Expr) -> torch.dtype:
```
No reason not to add type hints here, right?

Reply:
Thanks for the suggestion. Type hints are added.